azure-proto-compute/AvailabilitySetOperations.cs

```diff
@@ -7,21 +7,13 @@
 
 namespace azure_proto_compute
 {
-    public class AvailabilitySetOperations : ResourceOperationsBase<PhAvailabilitySet>
+    public class AvailabilitySetOperations : ResourceOperationsBase<AvailabilitySetOperations, PhAvailabilitySet>
     {
 
-        public AvailabilitySetOperations(ArmClientContext parent, TrackedResource context) : base(parent, context)
-        {
-        }
-        public AvailabilitySetOperations(ArmClientContext parent, ResourceIdentifier context) : base(parent, context)
-        {
-        }
-        public AvailabilitySetOperations(OperationsBase parent, TrackedResource context) : base(parent, context)
-        {
-        }
-        public AvailabilitySetOperations(OperationsBase parent, ResourceIdentifier context) : base(parent, context)
-        {
-        }
+        public AvailabilitySetOperations(ArmClientContext context, TrackedResource resource) : base(context, resource) { }
+
+        public AvailabilitySetOperations(ArmClientContext context, ResourceIdentifier id) : base(context, id) { }
```

Comment:
Do we still need this overload?

Reply:
The tracked resource one?  Yes its used by the Create method since you get a Model back and we need to pass that Model in here so its fullly filled out.

Reply:
No, I mean the ResourceIdentifier overload